problem-sets-competitive-programming.md

```diff
@@ -35,6 +35,7 @@
 * [LeetCode](https://leetcode.com)
 * [Microcorruption](https://microcorruption.com/login)
 * [Sphere Online Judge](http://www.spoj.com/contests)
+* [Techie Delight](https://www.techiedelight.com)
```

Comment:
not really competitive programming, and the quora link is the same content, don't you think?

Reply:
@eshellman okay I will remove the techie delight link. The previous quora link gets redirected to the current link I updated it too, but yes it's the same content.